### ISSUE TITLE
Rename "master" branch to "main" 

### ISSUE THREAD

**Issue by Julian88Tex** [kupferlauncher/kupfer#178]
Why: [theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main)
How: [gist.github.com/danieldogeanu/739f88ea5312aaa23180e162e3ae89ab](https://gist.github.com/danieldogeanu/739f88ea5312aaa23180e162e3ae89ab)
More context: [github/renaming](https://github.com/github/renaming)

**Comment by khurshid-alam** [comment]
Github doesn't change anything through. You can give any name to your branch like "prime".

**Comment by Julian88Tex** [comment]
> Github doesn't change anything through. You can give any name to your branch like "prime". 

@khurshid-alam sorry can you rephrase. Not sure I understand your comment.

**Comment by Julian88Tex** [comment]
@KarolBedkowski just checking it says completed but is that actually a "won't do"?
